app/javascript/controllers/audio_breakpoint_controller.js

```diff
@@ -30,10 +31,29 @@ export default class extends Controller {
   }
 
   startTimeValueChanged() {
+    if (this.hasInitialMarkerValue) {
+      const isChanged = this.startTimeValue !== this.initialMarkerValue.startTime
+
+      this.startTimeTarget.parentNode.classList.toggle("is-changed", isChanged)
```

Comment:
Hrmm... for me, I'm getting all orange highlighted fields all the time.  Even after clicking save or refreshing the page.

Reply:
Do you have a specific episode this is happening on?

Reply:
Found what was causing this. Fixed.